### PR TITLE
Update English quick start 

### PR DESCRIPTION
**Pull request by rayzhou2017**
- set email server for devops pipeline
- update out-of-scm pipeline tutorial - part I

### CODE REVIEW

File: content/v2.1/en/quick-start/bookinfo-canary.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
 ---
 title: "Managing Canary Release of Microservice App based on Istio"
-keywords: 'kubernetes, docker, helm, jenkins, istio, prometheus'
-description: ''
+keywords: 'kubesphere, kubernetes, docker, istio, canary release, jaeger'
+description: 'How to manage canary release of microservices using Istio platform'
 ---
 
-Istio’s service mesh provides the control necessary to manage traffic distribution with complete independence from deployment scaling. This allows for a simpler, yet significantly more functional, way to do canary test and rollout. **Canary release** provides canary rollouts, and staged rollouts with percentage-based traffic splits.
+[Istio](https://istio.io/)，as a open source service mesh, provides powerful traffic management which makes canary release of a microservice possible. **Canary release** provides canary rollouts, and staged rollouts with percentage-based traffic splits.
```
Comment (FeynmanZhou) [inline content/v2.1/en/quick-start/bookinfo-canary.md]: a open => an open source

File: content/v2.1/en/quick-start/jenkinsfile-out-of-scm.md [hunk 2]
```diff
@@ -252,97 +248,96 @@ Click **OK** to save the it.
...
 
 > Note: If the previous steps are running correctly, you can see that the pipeline has successfully run to the last stage in a few minutes. Since we set the review step and specify the account `project-admin` as the reviewer. Therefore, we need to switch using `project-admin` to manually review and approve it.
 
-4. Log out, and log in with account `project-admin`. Enter into the pipeline `graphical-pipeline` of the DevOps project that we used above. Drill into `Activity` to view the running status. You can see the pipeline has run to the `Deploy to DEV` stage. Click `Proceed` to approve its run.
+5.4. Log out, and log in with account `project-admin`. Enter into the pipeline `graphical-pipeline` of the DevOps project that we used above. Drill into `Activity` to view the running status. You can see the pipeline has run to the `Deploy to DEV` stage. Click `Proceed` to approve its run.
 
 ![Activity](https://pek3b.qingstor.com/kubesphere-docs/png/20200222170334.png)
 
-### View Pipeline
+### Step 5: View Pipeline
```
Comment (FeynmanZhou) [inline content/v2.1/en/quick-start/jenkinsfile-out-of-scm.md]: Step 5: => should be Step 6

File: content/v2.1/en/quick-start/jenkinsfile-out-of-scm.md [hunk 3]
```diff
@@ -1,104 +1,101 @@
...
 
-
 ![Advanced Settings](https://pek3b.qingstor.com/kubesphere-docs/png/20200222155944.png)
 
-### Editing pipeline
+### Step 4: Editing pipeline
 
 This pipeline consists of six stages, we will demonstrate steps and tasks in each stage.
 
-#### Stage 1: Pull Source Code (Checkout SCM)
```
Comment (FeynmanZhou) [inline content/v2.1/en/quick-start/jenkinsfile-out-of-scm.md]: The Stage 1 ~ 6 are related to the pipeline stage, it might be better to put them in order.

Comment (rayzhou2017) [reply content/v2.1/en/quick-start/jenkinsfile-out-of-scm.md]: It might cause user because of step and stage together here.

**Pull request merged**